nix/release.nix

```diff
@@ -3,4 +3,19 @@ let
 in {
   # Normal cabal build where Nix handles dependencies.
   vaultenv = pkgs.haskellPackages.callPackage ../vaultenv.nix {};
+
+  # Static package build
+  vaultenvStatic = (pkgs.pkgsStatic.haskellPackages.callPackage ../vaultenv.nix {
+    # Use non-static version of glibc locales, as the static version fails to build.
+    glibcLocales = pkgs.glibcLocales;
+  }).overrideAttrs (prev: {
+    # Disable the check phase, as hspec-discover will not run.
+    # TODO: investigate what's happening with hspec-discover.
```

Comment:
Do you still intend to debug the `TODO`s? Could you make issues otherwise?

Reply:
Both TODO's don't seem overly important.
The first because the tests are run elsewhere anywhere.
The second because it is just a slightly ugly, but still functional, solution.

Reply:
It doesn't pick up the tests when enabled.
Since the `pkgsStatic.haskellPackages` call is, in Falco's words, 'magic', I think I lack the knowledge to debug this.